source/elements/oneMKL/source/domains/spblas/data_types/format-descriptions.rst

```diff
@@ -0,0 +1,152 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_format_descriptions:
+
+Sparse storage formats
```

Comment:
Please let me know if you have made any changes in this file compared to the original description. If you have, I'll review it, but for now, I have skipped reviewing it.

Reply:
There are some minor changes: updated copyright date, formatted the indentation, renamed `nrows` and `ncols` with `num_rows` and `num_cols` to match with other functions, use preformatted text instead of bold.
Other than this only this commit has changed the content of this file: https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/96bee855d4da59333356e171c99f1d988e6239d9